catalog/core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -84,6 +84,8 @@
 
     private static final String BYTES_TO_SKIP = "BytesToSkip";
 
+    private static final String CONTENT_RANGE_HEADER = "Content-Range";
```

Comment:
You might be able to use this constant from Guava: https://google.github.io/guava/releases/19.0/api/docs/com/google/common/net/HttpHeaders.html#CONTENT_RANGE